docs/walkthrough/walkthrough_data.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Walkthrough data
+summary: Example data for walkthrough and testing
+permalink: walkthrough_data.html
+datatable: true
+---
+{% assign walkthrough_folder = "walkthrough/data/" %}
+
+This page describes the dataset that is used in the 
```

Comment:
It was in purpose, because it is only a set of 203 paired-end reads but in different formats.